lib/rubocop/cop/style/return_nil.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop enforces consistency between 'return nil' and 'return'.
+      #
+      # Supported styles are: return, return_nil.
+      #
+      # @example
+      #   # bad if EnforcedStyle is return, good if EnforcedStyle is return_nil
```

Comment:
Please add a section for each `EnforcedStyle`, as it's easier to understand. 🙂 Also, the example right now would cause another RuboCop offense (redundant return.) It would be nice if the examples were non-offending code. Metasyntactic variables are a conventional way to indicate that a name doesn't matter, i.e. `abcd` -> `foo`.

Grouping example:

```
# @example
#
#   # EnforcedStyle: return (default)
#
#   # bad
#   def foo
#     return nil
#   end
#
#   # good
#   def foo
#     return
#   end
```